common/common.go

```diff
@@ -51,6 +53,10 @@ const (
 	AttributeDroppedEventsCount             = "otel.span.dropped_events_count"
 	AttributeDroppedLinkAttributesCount     = "otel.link.dropped_attributes_count"
 	AttributeDroppedLinksCount              = "otel.span.dropped_links_count"
+	AttributeResourceAttribute              = "otel.resource.attributes"
```

Comment:
When should `otel.resource.attributes` be used? I don't see anywhere where we are currently using its associated `otel.resource.dropped_attributes_count` name either.